src/ImageSharp/MetaData/Profiles/Exif/ExifProfile.cs

```diff
@@ -239,7 +235,8 @@ public void SetValue(ExifTag tag, object value)
         /// Converts this instance to a byte array.
         /// </summary>
         /// <param name="includeExifIdCode">Indicates, if the Exif ID code should be included.
-        /// This Exif ID code should not be included in case of PNG's. Defaults to true.</param>
+        /// The Exif Id Code is part of the JPEG APP1 segment. This Exif ID code should not be included in case of PNG's.
+        /// Defaults to true.</param>
         /// <returns>The <see cref="T:byte[]"/></returns>
         public byte[] ToByteArray(bool includeExifIdCode = true)
```

Comment:
Pass through the code as a `ReadonlySpan<bye>` copying that to the output array.